arbeitszeit_flask/translations/de/LC_MESSAGES/messages.po

```diff
@@ -1700,26 +1734,38 @@ msgstr "Ungültige Plan-ID."
 msgid "Invalid cooperation ID."
 msgstr "Ungültige Kooperations-ID."
 
-#: arbeitszeit_web/www/presenters/accept_cooperation_request_presenter.py:21
+#: arbeitszeit_web/www/controllers/select_productive_consumption_controller.py:46
+msgid "The provided plan ID is not a valid UUID."
+msgstr "Die angegebene Plan-ID ist keine gültige UUID."
+
+#: arbeitszeit_web/www/controllers/select_productive_consumption_controller.py:66
+msgid "The provided amount is not a valid integer."
```

Comment:
from the german translation, shouldn't this be "The provided amount is not a valid number."?

Reply:
I think the correct translation for "ganze Zahl" is "integer", and vice versa. https://de.pons.com/%C3%BCbersetzung-2/englisch-deutsch/integer